GCTC-NTGC/gc-digital-talent
# Bug - Fix site webmanifest icon paths

Resolves #4078.

## Bonus
Adds missing `indigenousapprenticeship` icons referenced in webmanifest.

## Screenshots
![Screen Shot 2022-11-16 at 14 56 04](https://user-images.githubusercontent.com/3046459/202281802-0f9bd036-1575-470a-ac28-2d613b0173bb.png)
![Screen Shot 2022-11-16 at 14 56 18](https://user-images.githubusercontent.com/3046459/202281803-621ca277-169a-4c2f-815f-c6412d907ad5.png)
![Screen Shot 2022-11-16 at 14 56 57](https://user-images.githubusercontent.com/3046459/202281804-1e4e7138-f984-4a3f-b881-e5b89b677af7.png)

## Steps to test
1. Build `npm run production --workspace=talentsearch`
2. Build `npm run production --workspace=admin`
3. Build `npm run production --workspace=indigenousapprenticeship`
4. Visit each site
5. Open Chrome DevTools > Application > Manifest > Icons
6. Verify icons are correct
